alecs-theme/src/WEB-INF/src/resources-importer/journal/structures/Service.json

```diff
@@ -0,0 +1,191 @@
+{
+    "availableLanguageIds": [
+        "en_US"
+    ],
+    "defaultLanguageId": "en_US",
+    "fields": [
+        {
+            "label": {
+                "en_US": "Background Color"
+            },
+            "predefinedValue": {
+                "en_US": ""
+            },
+            "style": {
+                "en_US": ""
+            },
+            "tip": {
+                "en_US": ""
+            },
+            "dataType": "color",
+            "fieldNamespace": "ddm",
+            "indexType": "keyword",
+            "localizable": true,
+            "name": "BackgroundColor",
+            "readOnly": false,
+            "repeatable": false,
+            "required": true,
+            "showLabel": true,
+            "type": "ddm-color"
+        },
```

Comment:
Are we expecting a hex value?  RGBA? word?  Maybe use a select list to predefine and limit the available options for the end user?  Add a predefined value, tip or some description so the end user knows what to do.

Reply:
Predefined color set and tip added. The default color datatype allows for hex or RGB via gui:

![image](https://user-images.githubusercontent.com/28070886/44360748-abc66b00-a470-11e8-8e48-a3c19a8a6b93.png)